### PR TITLE
Rad command uses bindir to execute

### PR DESCRIPTION
**Pull request by geigerzaehler**
The `rad` command now uses `getBinDir` instead of `findExecutable` to get the path for the subcommands. This allows us to install the subcommands into a folder not contained in `PATH`

We also rename `radicle-daemon` to `rad-daemon-radicle` and `rad-ipfs-daemon` to `rad-daemon-ipfs`.

### CODE REVIEW

File: exe/Rad.hs [hunk 1]
```diff
@@ -50,18 +51,15 @@ builtinCommands :: [String]
...
-            Right xs                -> xs ++ builtinCommands)
+-- | Returns list of Radicle sub-commands located in separate binaries.
+-- The first item is the command name, the second the absolute location
+-- of the executable.
+getSubCommands :: IO [(String, FilePath)]
+getSubCommands = do
+    searchDir <- getBinDir
+    files <- listDirectory searchDir
+    let commands = mapMaybe (stripPrefix radPrefix) files
+    pure $ [ (command, searchDir </> radPrefix <> command) | command <- commands]
```
Comment (cloudhead) [inline exe/Rad.hs]:
```suggestion
    pure [(command, searchDir </> radPrefix <> command) | command <- commands]
```

File: exe/Rad.hs [hunk 2]
```diff
@@ -50,18 +51,15 @@ builtinCommands :: [String]
...
-    -> IO [String] -- ^ Available commands
-availableCommands dirs =
-    concat <$> forM dirs (\dir -> do
-        result <- try $ do
-            xs <- listDirectory dir
-            pure $ [drop (length radPrefix) x | x <- xs, toS radPrefix `isPrefixOf` x]
-        pure $ case result of
-            Left (_ :: IOException) -> builtinCommands
-            Right xs                -> xs ++ builtinCommands)
+-- | Returns list of Radicle sub-commands located in separate binaries.
```
Comment (cloudhead) [inline exe/Rad.hs]:
```suggestion
-- | Returns list of Radicle sub-commands located in separate directories.
```

Comment (cloudhead) [reply exe/Rad.hs]: I'm guessing..

Comment (geigerzaehler) [reply exe/Rad.hs]: I meant something different. I rewrote the docs to make it clearer.

**Pull request merged**